book_source/Makefile

```diff
@@ -13,4 +11,5 @@ deploy:
 	./deploy.sh
 	
 pdf:
+	./check_bkd_pkg.R
```

Comment:
why not add build to pdf:?

Reply:
as in `pdf: build`? I assume because the render_book calls are doing different things such that it's not necessarily desirable to wait for both when calling one.

But now that you mention it I should have done `build: bkdcheck` and `pdf: bkdcheck` instead of removing the rule.. Will change.